src/main/java/com/laytonsmith/core/AliasCore.java

```diff
@@ -344,121 +326,218 @@ public final void reload(MCPlayer player, String[] settings, boolean firstLoad)
 				}
 			}
 
-			if(!Prefs.isInitialized()) {
-				Prefs.init(prefFile);
+			// Get the default files (main.ms, aliases.msa, auto_include.ms)
+			try {
+				String main = file_get_contents(mainFile.getAbsolutePath());
```

Comment:
We should probably remove these methods, and use the normal utility methods. What is this, [PHP](https://www.php.net/manual/en/function.file-get-contents.php)?!?